apps/cpp_rpc/rpc_server.cc

```diff
@@ -217,10 +217,10 @@ class RPCServer {
    * \param opts Parsed options for socket
    * \param ping_period Timeout for select call waiting
    */
-  void AcceptConnection(TrackerClient* tracker, 
+  void AcceptConnection(TrackerClient* tracker,
```

Comment:
if spaces are ending up in the source code, should we turn on the linter

Reply:
this is because apps are not covered by the linter as oppose the `src/`